crates/trie/common/src/hashed_state.rs

```diff
@@ -324,17 +324,8 @@ impl HashedPostState {
 
     /// Converts hashed post state into [`HashedPostStateSorted`].
     pub fn into_sorted(self) -> HashedPostStateSorted {
```

Comment:
You can replace the body of this method with just `self.drain_into_sorted()`, like was done with TrieUpdatesSorted, to reduce duplicate code